vendor/modules.txt

```diff
@@ -885,7 +885,7 @@ k8s.io/kube-openapi/pkg/spec3
 k8s.io/kube-openapi/pkg/util/proto
 k8s.io/kube-openapi/pkg/validation/spec
 # k8s.io/mount-utils v0.31.0-alpha.0
```

Comment:
Hmm not related to this PR but this is not right, why are we depending on the alpha version.

Reply:
It comes from dependabot https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/commit/cde24e7aad20a314d1c83eeedb67debbc3f1e929 Seems like we should revert it?